src/proto/streams/send.rs

```diff
@@ -81,28 +81,61 @@ where
         Ok(())
     }
 
+    /// Send an RST_STREAM frame
+    ///
+    /// # Arguments
+    /// + `clear_queue`: if true, all pending outbound frames will be cleared,
+    ///    if false, the RST_STREAM frame will be appended to the end of the
+    ///    send queue.
```

Comment:
I think this all works, but if @carllerche wants to glance over, that wouldn't hurt.